NEWS.md

```diff
@@ -87,6 +87,8 @@ Standard library changes
 
 #### LinearAlgebra
 
+* `rank` can now take a QRPivoted matrix ([#53214]) to allow rank computation on QR-factorized matrices.
```

Comment:
The news item should cross reference the PR, not the issue.

Reply:
My bad, it should be ok now